src/vue/navigation/Navbar.vue

```diff
@@ -0,0 +1,411 @@
+          </template>
+        </div>
+      </div>
+
+      <div
+        class="navbar__user-card"
+        :class="{ 'navbar__user-card--active': isUserCardOpen }"
+      >
+        <div class="navbar__user-card-ctn">
+          <div class="navbar__user-card-content">
+            <div class="navbar__user-avatar">
+              {{ walletEmail.substr(0, 1).toUpperCase() }}
+            </div>
+            <div class="navbar__user-card-info">
+              <p class="navbar__user-card-name">
+                {{ walletEmail }}
+              </p>
+              <div class="navbar__user-card-status">
+                <template v-if="accountTypeI === ACCOUNT_TYPES.notVerified">
+                  {{ 'navbar.account-type-unverified' | globalize }}
+                </template>
+                <template v-else-if="accountTypeI === ACCOUNT_TYPES.individual">
+                  {{ 'navbar.account-type-general' | globalize }}
+                </template>
+                <template v-else-if="accountTypeI === ACCOUNT_TYPES.syndicate">
+                  {{ 'navbar.account-type-corporate' | globalize }}
+                </template>
+              </div>
+              <button
+                v-ripple="'rgba(255, 255, 255, .2)'"
+                @click="goKyc"
+                class="navbar__user-card-account-btn"
+              >
+                {{ 'navbar.my-account' | globalize }}
+              </button>
+            </div>
+          </div>
+          <div class="navbar__user-actions">
+            <button
+              v-ripple
+              class="navbar__user-action"
+              @click="goSettings"
+            >
+              {{ 'navbar.settings' | globalize }}
+            </button>
+            <button
+              v-ripple
+              class="navbar__user-action"
+              @click="signOut"
+            >
+              {{ 'navbar.signout' | globalize }}
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </nav>
+</template>
+
+<script>
+import { vuexTypes } from '@/vuex'
+import { mapActions, mapGetters } from 'vuex'
+import { vueRoutes } from '@/vue-router'
+import { ACCOUNT_TYPES } from '@/js/const/xdr.const'
+import { handleClickOutside } from '@/js/helpers/handle-click-outside'
+
+export default {
+  name: 'root-navbar',
+
+  data: () => ({
+    isUserCardOpen: false,
+    ACCOUNT_TYPES
+  }),
+  computed: {
+    ...mapGetters({
+      walletEmail: `new-wallet/${vuexTypes.walletEmail}`,
+      accountType: `new-account/${vuexTypes.accountType}`,
+      accountTypeI: `new-account/${vuexTypes.accountTypeI}`
+    })
+  },
+  watch: {
+    isUserCardOpen (value) {
+      if (value) {
+        handleClickOutside(
+          'navbar__user-card',
+          this.toggleUserCardVisibility
+        )
+      }
+    }
+  },
+  methods: {
+    ...mapActions([
+      vuexTypes.LOG_OUT
+    ]),
+    toggleUserCardVisibility () {
+      this.isUserCardOpen = !this.isUserCardOpen
+    },
+    signOut () {
```

Comment:
to avoid uppercase letters, we can pass the object to `mapActions`:

```
...mapActions({
  logOut: vuexTypes.LOG_OUT
})
```

and use consistent names, I mean if we call the action `log out`, we don't call it `sign out` anywhere